packages/dht/src/connection/WebRTC/NodeWebRtcConnection.ts

```diff
@@ -0,0 +1,201 @@
+import { IWebRtcConnection, Event as IWebRtcEvent } from './IWebRtcConnection'
+import { ConnectionType, IConnection, Event as ConnectionEvent, } from '../IConnection'
+import { ConnectionID } from '../../types'
+import { PeerDescriptor } from '../../proto/DhtRpc'
+import EventEmitter = require('events')
+import nodeDatachannel, { DataChannel, DescriptionType, PeerConnection } from 'node-datachannel'
+import { PeerID } from '../../helpers/PeerID'
+import { Logger } from '../../helpers/Logger'
+import { IWebRtcCleanUp } from './IWebRtcCleanUp'
+
+const logger = new Logger(module)
+
+export const WebRtcCleanUp = new class implements IWebRtcCleanUp {
+    cleanUp(): void {
+        nodeDatachannel.cleanup()
+    }
+}
+
+export class NodeWebRtcConnection extends EventEmitter implements IConnection, IWebRtcConnection {
```

Comment:
```suggestion
export class NodeWebRtcConnection extends EventEmitter3<IWebRtcEvent> implements IConnection, IWebRtcConnection {
```

This adds type safety, and requires `eventemitter3` package (`import EventEmitter3 from 'EventEmitter3'`). Could apply this pattern for all classes which extends an event emitter.

Similar pattern is used in`StreamrClient`:
https://github.com/streamr-dev/network-monorepo/blob/main/packages/client/src/events.ts

Reply:
https://linear.app/streamr/issue/NET-872/technical-debt-net-763